uvm/impl/com/untangle/uvm/MailSenderImpl.java

```diff
@@ -608,7 +617,7 @@ private void writeConfiguration()
 
             int targetPort = 0;
             if (settings.getSendMethod() == MailSettings.SendMethod.CUSTOM) targetPort = settings.getSmtpPort();
-            if (settings.getSendMethod() == MailSettings.SendMethod.RELAY) targetPort = STUNNEL_RELAY_PORT;
+            //if (settings.getSendMethod() == MailSettings.SendMethod.RELAY) targetPort = STUNNEL_RELAY_PORT;
```

Comment:
I think it will best for us to just leave the code there instead of commenting it.